TurtleArtActivity.py

```diff
@@ -866,7 +866,6 @@ def _setup_toolbar(self):
         self.palette_toolbar_button = ToolbarButton(
             page=self._palette_toolbar, icon_name='palette')
 
-        self._help_button = HelpButton(self)
```

Comment:
I don't understand why you have regressed the code at this point.  See https://github.com/sugarlabs/turtleart-activity/blob/master/TurtleArtActivity.py#L801